src/traverse/node-path.js

```diff
@@ -34,9 +34,62 @@ class NodePath {
   }
 
   /**
-   * Inserts a node into a collection. By default child nodes
-   * are supposed to by under `expressions` property. An explicit
-   * property can be passed.
+   * sets a node into a children collection or the single child.
+   * By default child nodes are supposed to be under `expressions` property.
+   * An explicit property can be passed.
+   *
+   * @param Object node - a node to set into a collection or as single child
+   * @param number index - index at which to set
+   * @param string property - name of the collection or single property
+   */
+  setChild(node, index = null, property = null) {
+
+    if (index != null) {
+      if (!property) property = "expressions";
```

Comment:
nit: single quotes for strings per code style, and wrap to full block (even if one expression)

```js
if (!property) {
  property = 'expressions';
}
```

Also, `expressions` appears already in many methods. We can put it into a constant at the top of the file, and reuse in all other methods to avoid typos.

```js

const DEFATUL_COLLECTION_PROP = 'expressions';

...

if (!property) {
  property = DEFATUL_COLLECTION_PROP;
}
```